core/spec/models/spree/image_spec.rb

```diff
@@ -9,6 +9,22 @@
     let(:default_style) { :product }
   end
 
+  describe 'validations' do
+    let(:valid_image_file) { File.open(File.join('lib', 'spree', 'testing_support', 'fixtures', 'blank.jpg')) }
+    let(:invalid_image_file) { File.open(File.join('lib', 'spree', 'testing_support', 'fixtures', 'file.txt')) }
+    let(:image) { create(:image) }
```

Comment:
I think that using this syntax the specs should work with both adapters:

`subject { create(:image, attachment: valid_image_file) }`